src/zjs_spi.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2017, Intel Corporation.
+
+#ifndef __zjs_spi_h__
+#define __zjs_spi_h__
+
+#include "jerryscript.h"
+
+enum spi_topology {
```

Comment:
It seems there's no need for these to be public, so maybe move these to the source file instead.